internal/store/store_test.go

```diff
@@ -444,6 +446,7 @@ func (suite *storeTest) TestNewSourceConfigWithMirrorlist() {
 	myRepoConfig := rpmmd.RepoConfig{
 		Name:       "testRepo",
 		Mirrorlist: "testURL",
+		IgnoreSSL:  false,
```

Comment:
I'm wondering why was this needed, since in this commit (https://github.com/osbuild/osbuild-composer/pull/3228/commits/d34fe71bdded6dd6f3a46c8111ba6da0bb375062) I don't see any change related to `IgnoreSSL` struct member 🤔

Reply:
Good catch! I think I made `IgnoreSSL` a pointer and then undid it - but that's my only guess.